parsing/g_prim.mlg

```diff
@@ -61,12 +61,12 @@ GRAMMAR EXTEND Gram
   pattern_identref:
     [ [ id = pattern_ident -> { CAst.make ~loc id } ] ]
   ;
-  var: (* as identref, but interpret as a term identifier in ltac *)
-    [ [ id = ident -> { CAst.make ~loc id } ] ]
-  ;
   identref:
     [ [ id = ident -> { CAst.make ~loc id } ] ]
   ;
+  hyp: (* as identref, but interpreted as an hypothesis in tactic notations *)
+    [ [ id = identref -> { id } ] ]
+  ;
```

Comment:
Maybe slide up above identref, next to the deletion.

Reply:
I moved it because it now depends on `identref`. This part of code will anyway be reorganized in #11842.